packages/components/src/components/popup/popup.tsx

```diff
@@ -76,7 +75,7 @@ const Popup = ({
                     </MobileFullPageModal>
                 </div>
             </PopupContext.Provider>,
-            document.getElementById('popup_root')
+            document.getElementById('popup_root')!
```

Comment:
Just for my understanding what is this sign `!` for?

Reply:
it means that the element will always exist in the document. Something like null-check. you can check on this [link](https://stackoverflow.com/questions/63520680/argument-of-type-htmlelement-null-is-not-assignable-to-parameter-of-type-el)

Reply:
It's non null assertion. 
check this: https://stackoverflow.com/questions/55588968/type-error-object-is-possibly-null-ts2531-for-window-document